src/charm.py

```diff
@@ -133,7 +145,25 @@ def _reconcile(self, _: ops.EventBase) -> None:
             self.container.start("exporter")
         else:
             self.container.stop("exporter")
-        self.unit.status = ops.ActiveStatus()
+        deadline = time.time() + 120
+        self.unit.status = ops.WaitingStatus("waiting for penpot services")
+        while time.time() < deadline:
+            if self._check_penpot_backend_ready():
+                self.unit.status = ops.ActiveStatus()
+                return
+            time.sleep(3)
+        self.unit.status = ops.BlockedStatus("timeout waiting for penpot services")
```

Comment:
We can't use the Error state as it will cause an error loop for [reasons pointed out by Javier](https://github.com/canonical/penpot-operator/pull/9#discussion_r1726497033). This is just a quality-of-life feature, as Penpot will not be available for a few seconds after the service starts, so the wait here will inform the user that Penpot is currently starting. Also, this makes integration tests easier as a `wait_for_idle` will automatically wait until the Penpot service is ready.

There's already a Pebble health check for this, so Kubernetes will kill this pod a few minutes after this message shows up.

Reply:
Continuing the discussion in the pointed comment of Javier